sda-download/dev_utils/config.yaml

```diff
@@ -44,6 +44,10 @@ c4gh:
   passphrase: "oaagCP1YgAZeEyl2eJAkHv9lkcWXWFgm"
   filepath: "./dev_utils/c4gh.sec.pem"
 
+grpc:
+    host: "reencrypt"
+    port: "50051"
```

Comment:
Credentials are missing here

Reply:
see https://github.com/neicnordic/sensitive-data-archive/pull/761#discussion_r1535712455

Reply:
The mTLS is enabled in the commit 9f549c32